protocol/read-write-resource/method-not-allowed.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Respond with 405 for non-existent method
+
+  Background: Set up clients and paths
+    * def testContainer = rootTestContainer.createContainer()
```

Comment:
```suggestion
    * def testContainer = rootTestContainer.reserveContainer()
```
If you are going to also create a resource, you only need to reserve the container as the resource creation will ensure it exists (assuming the server is conforming to containment rules). This is the pattern I have used elsewhere - do you see any concerns with that pattern?

Reply:
No, but I only want to say `DAHU` on that resource (whatever that might mean), not create it. :-)